builder/builder.go

```diff
@@ -277,18 +302,19 @@ func (b *Builder) OnPayloadAttribute(attrs *types.BuilderPayloadAttributes) erro
 	b.slotMu.Lock()
 	defer b.slotMu.Unlock()
 
-	if b.slot != attrs.Slot {
-		if b.slotCtxCancel != nil {
-			b.slotCtxCancel()
-		}
+	// Forcibly cancel previous building job, build on top of reorgable blocks as this is the behaviour relays expect.
```

Comment:
Can you please explain why "buidling on multiple tips" is no longer necessary?

Reply:
It is, it just doesn't work with how relays are set up. This will come back shortly after Capella.